humagne.yaml

```diff
@@ -110,7 +110,7 @@ packages:
       - gaussian@g16-A.03
       - gurobi@8.1.1
       - smr@2017.06
-      - cfdplusplus@16.1
+      - cfdplusplus@19.1
```

Comment:
As we are probably not going to remove version 16.1 (let me know otherwise), I suggest we add version 19.1 but do not remove 16.1 from the list. (To more closely reflect reality.)